### ISSUE TITLE
"Box" Is not part of "ChildComponents" as documentation states

### ISSUE THREAD

**Issue by IanKrieger** [open-olive/loop-development-kit#312]
In the documentation found here: https://open-olive.github.io/loop-development-kit/ldk/javascript/globals.html#childcomponents it states that `Box` is part of the `ChildComponents`

However in the `node_modules/@oliveai/ldk/src/whisper/types.ts` that is not the case.
```
export type ChildComponents =
  | Button
  | Checkbox
  | Divider
  | Email
  | Link
  | ListPair
  | Markdown
  | Message
  | NumberInput
  | Password
  | RadioGroup
  | Select
  | Telephone
  | TextInput;
```

Please update the `ChildComponents` to reflect the documentation.

**Comment by DanielWagersOlive** [comment]
Hi @IanKrieger!  Thank you for noting the issue.  This should be fixed in release `v3.2.0`

**Comment by CandiceCLB** [comment]
This has been resolved with V3.2.0.
